site/src/analytics-addons.ts

```diff
@@ -0,0 +1,672 @@
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/ssh/icon.png",
+  },
+
+  a0d7b954_tailscale: {
+    name: "Tailscale",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/tailscale/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/tailscale/icon.png",
+  },
+  a0d7b954_tasmoadmin: {
+    name: "TasmoAdmin",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/tasmoadmin/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/tasmoadmin/icon.png",
+  },
+  a0d7b954_tautulli: {
+    name: "Tautulli",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/tautulli/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/tautulli/icon.png",
+  },
+  a0d7b954_thelounge: {
+    name: "The Lounge",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/thelounge/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/thelounge/icon.png",
+  },
+  a0d7b954_tor: {
+    name: "Tor",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/tor/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/tor/icon.png",
+  },
+  a0d7b954_traccar: {
+    name: "Traccar",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/traccar/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/traccar/icon.png",
+  },
+  a0d7b954_unifi: {
+    name: "UniFi Network Application",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/unifi/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/unifi/icon.png",
+  },
+  a0d7b954_vscode: {
+    name: "Studio Code Server",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/vscode/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/vscode/icon.png",
+  },
+  a0d7b954_wireguard: {
+    name: "WireGuard",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/wireguard/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/wireguard/icon.png",
+  },
+  a0d7b954_zerotier: {
+    name: "ZeroTier One",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/zerotier/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/zerotier/icon.png",
+  },
+  a0d7b954_zwavejs2mqtt: {
+    name: "Z-Wave JS to MQTT",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/zwavejs2mqtt/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/zwavejs2mqtt/icon.png",
+  },
+};
```

Comment:
Can we add the ESPHome Repo.

Reply:
We can do that when it's shipping with the supervisor.